src/wp-includes/class-wp-query.php

```diff
@@ -1300,10 +1319,12 @@ public function parse_tax_query( &$q ) {
 
 		if ( '' !== $q['tag'] && ! $this->is_singular && $this->query_vars_changed ) {
 			if ( str_contains( $q['tag'], ',' ) ) {
+				// @todo Handle normalizing `tag` query string.
```

Comment:
@peterwilsoncc do you intend to keep this comment, or is this something that still needs to be implemented before this is ready?

Reply:
I intend to keep it and have it as a task for later. This block of code does some strange things so I decided to leave it for a follow up.